util/Migrator/DbScripts/2023-01-24_00_AutoscalingProviderOrgFixes.sql

```diff
@@ -0,0 +1,71 @@
+-- SG-992 changes: add planType to provider orgs
+CREATE OR ALTER  VIEW [dbo].[ProviderUserProviderOrganizationDetailsView]
+AS
+SELECT
+    PU.[UserId],
+    PO.[OrganizationId],
+    O.[Name],
+    O.[Enabled],
+    O.[UsePolicies],
+    O.[UseSso],
+    O.[UseKeyConnector],
+    O.[UseScim],
+    O.[UseGroups],
+    O.[UseDirectory],
+    O.[UseEvents],
+    O.[UseTotp],
+    O.[Use2fa],
+    O.[UseApi],
+    O.[UseResetPassword],
+    O.[SelfHost],
+    O.[UsersGetPremium],
+    O.[UseCustomPermissions],
+    O.[Seats],
+    O.[MaxCollections],
+    O.[MaxStorageGb],
+    O.[Identifier],
+    PO.[Key],
+    O.[PublicKey],
+    O.[PrivateKey],
+    PU.[Status],
+    PU.[Type],
+    PO.[ProviderId],
+    PU.[Id] ProviderUserId,
+    P.[Name] ProviderName,
+    O.[PlanType] -- new prop
+FROM
+    [dbo].[ProviderUser] PU
+    INNER JOIN
+    [dbo].[ProviderOrganization] PO ON PO.[ProviderId] = PU.[ProviderId]
+    INNER JOIN
+    [dbo].[Organization] O ON O.[Id] = PO.[OrganizationId]
+    INNER JOIN
+    [dbo].[Provider] P ON P.[Id] = PU.[ProviderId]
+    GO
+
+        
+-- Refresh view metadata
+IF OBJECT_ID('[dbo].[ProviderUserProviderOrganizationDetailsView]') IS NOT NULL
```

Comment:
@r-tome , I just realized that I put the view name here instead of a stored procedure that uses the view. So, do you normally just search in code for references to the view and identify N stored procedures and have an `EXECUTE sp_refreshsqlmodule` for each stored proc that uses the view?

Reply:
I've actually only included that refresh metadata script once and that is exactly what I did!

Reply:
@r-tome , you are saying that you targeted the view? Or you went and looked up each stored proc that targeted the view and refreshed them?